### PR TITLE
fix: dashboard cm include exclude fix

### PR DESCRIPTION
**Pull request by rish2320**
# Description
Added a flag in dashboard-cm for include/exclude git commits.

Fixes # [3117](https://dev.azure.com/DevtronLabs/Devtron/_boards/board/t/OSS%20adoption/Stories)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Tested all the cases manually.

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/common/GitInfoMaterial.tsx [hunk 1]
```diff
@@ -288,9 +288,9 @@ export default function GitInfoMaterial({
                     >
                         {renderBranchChangeHeader(selectedMaterial)}
                         {!selectedMaterial.isRepoError && !selectedMaterial.isBranchError && (
-                            <div className="flex right mr-20">
+                            <div className={`flex right ${!window._env_.HIDE_EXCLUDE_INCLUDE_GIT_COMMITS && "mr-20"}`}>
```
Comment (milindk8) [inline src/components/common/GitInfoMaterial.tsx:291]: create a const for it as it is common

File: src/components/material/MaterialView.tsx [hunk 2]
```diff
@@ -161,7 +161,7 @@ export class MaterialView extends Component<MaterialViewProps, MaterialViewState
 
     regexInfoSteps = (): JSX.Element => {
         return (
-            
+
```
Comment (milindk8) [inline src/components/material/MaterialView.tsx:164]: remove this space

Comment (rish2320) [reply src/components/material/MaterialView.tsx]: done

File: src/components/common/GitInfoMaterial.tsx (hunk 1 above)
Comment (rish2320) [reply src/components/common/GitInfoMaterial.tsx]: done

File: src/components/common/GitInfoMaterial.tsx [hunk 3]
```diff
@@ -279,6 +279,7 @@ export default function GitInfoMaterial({
     function renderMaterialHistory(selectedMaterial: CIMaterialType) {
         let anyCommit = selectedMaterial.history?.length > 0
         const isWebhook = selectedMaterial.type === SourceTypeMap.WEBHOOK
+        const excludeIncludeEnv = window._env_.HIDE_EXCLUDE_INCLUDE_GIT_COMMITS
```
Comment (milindk8) [inline src/components/common/GitInfoMaterial.tsx:282]: Add not operation here

**Review by milindk8 — APPROVED** [review]

**Review by vivek-devtron — APPROVED** [review]

File: src/components/common/GitInfoMaterial.tsx (hunk 3 above)
Comment (rish2320) [reply src/components/common/GitInfoMaterial.tsx]: done

**Pull request merged**